.github/workflows/conformance-ingress.yaml

```diff
@@ -417,6 +417,17 @@ jobs:
             -enable-http-debug \
             -stop-on-failure
 
+      - name: Run basic CLI tests
+        shell: bash
+        run: |
+          mkdir -p cilium-junits
+          cilium connectivity test --include-unsafe-tests --collect-sysdump-on-failure \
+            --sysdump-hubble-flows-count=1000000 --sysdump-hubble-flows-timeout=5m \
+            --sysdump-output-filename "cilium-sysdump-${{ matrix.name }}-<ts>" \
+            --junit-file "cilium-junits/${{ env.job_name }} (${{ join(matrix.*, ', ') }}).xml" \
+            --junit-property github_job_step="Run tests (${{ join(matrix.*, ', ') }})" \
```

Comment:
As per comment before, should be `github_job_step="Run basic CLI tests (${{ matrix.name }})"